src/test/javascript/spec/component/admin/user-management.component.spec.ts

```diff
@@ -346,4 +346,214 @@ describe('UserManagementComponent', () => {
         comp.selectAllRoles();
         expect(comp.filters.authorityFilter).toEqual(new Set(comp.authorityFilters));
     });
+
+    it('should delete all selected users', () => {
+        const deleteSpy = jest.spyOn(userService, 'deleteUsers').mockReturnValue(of());
+
+        // users
+        const users = [1, 2, 3].map((id) => {
+            const user = new User();
+            user.login = id.toString();
+            return user;
+        });
+
+        comp.selectedUsers = [users[0], users[1]];
+
+        comp.deleteAllSelectedUsers();
+        expect(deleteSpy).toHaveBeenCalledOnce();
+        expect(deleteSpy).toHaveBeenCalledWith([users[0].login, users[1].login]);
+    });
+
+    it('should add and remove user from selected users', () => {
+        // user
+        const user = new User();
+        user.login = '1';
+
+        expect(comp.selectedUsers).toEqual([]);
+        comp.toggleUser(user);
+        expect(comp.selectedUsers).toEqual([user]);
+        comp.toggleUser(user);
+        expect(comp.selectedUsers).toEqual([]);
+    });
+
+    it('should return number of applied filters', () => {
+        comp.filters = new UserFilter();
+        expect(comp.filters.numberOfAppliedFilters).toEqual(0);
```

Comment:
```suggestion
        expect(comp.filters.numberOfAppliedFilters).toBe(0);
```
Compare primitive types with toBe